js/dashboard.js

```diff
@@ -33,20 +33,27 @@
 	}, {
 		low: 0,
 		showArea: true,
-		fullWidth: true,
+		fullWidth: ( data.length < 90 ),
+		width: ( data.length < 90 ? null : 5 * data.length ),
```

Comment:
I've had a look at the changes several times yet while updating to current master and had my cursor on the _Approve_ button already...

Now I've found one border case where we waste some space:

![statify_101_wastedspace](https://user-images.githubusercontent.com/12963621/79685817-dbc4a380-823b-11ea-9ded-b4915a853518.png)

Anything against a dynamic limit instead of "90 points"?

```javascript
	if ( jQuery( '#statify_chart' ).width() < data.length * 5 ) {
		fullWidth = false;
	}

	/* ... */

		fullWidth: fullWidth,
		width: ( fullWidth ? null : 5 * data.length ),
```

?